.github/workflows/test.yaml

```diff
@@ -282,6 +282,7 @@ jobs:
   build_wheels_linux:
     container: quay.io/pypa/manylinux2014_x86_64:latest
     env:
+      PANTS_CA_CERTS_PATH: /opt/_internal/certs.pem
```

Comment:
See https://github.com/pantsbuild/pex/issues/1674. tl;dr: Pants strips `SSL_CERT_FILE` from the env, so when Pex's downloader tries to load SSL we fail to find a certificate. That doesn't happen when Pex downloads via pip because pip uses `certifi` as a fallback. I'll soon add this fallback to Pex too, then delete this line.

Reply:
I guess that's fine, but also unless every vertical Pants supports comes with infra to embed its own root certs, there will be similar issues. And having every vertical embed its own root cert bundle of varying birth date & pedigree is ... clearly not awesome.

Reply:
I'm not following that - my (admittedly naive!) understanding is that we'll default to `certifi`—which provides root certs. If that doesn't work for you, then you can set Pant's `[GLOBAL].ca_certs_path`.

Why would every vertical need to embed its own root certs?

Reply:
Stepping back up from the weeds of this PR: If you're a Pants user and you're using Pants to do `<vertical>` for you, then if that vertical does not have tools Pants uses that embed their own certs and those tools are run via Pants in a container with standard env vars not set or not plumbed through Pants, you'll have this same issue all over again in that other vertical. So you get a picture of ~O(10) verticals each with their own embdded root certs of varying age and content (for those that do embed). It seems like the only sane thing is to have a single root cert set and plumb that through to all tools. You can then keep that cert set up2date like you do the rest of the packages on your machine.

So, yeah, its convenient Pip and now Pex will be magically "saving you from a headache" - but you'll just be avoiding the truth. And a headache will turn into a bigger problem further down the line.